contents/handbook/engineering/incidents.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Handling an incident
+sidebar: Handbook
+showTitle: true
+---
+
+Incidents are going to happen.
+
+## When to raise an incident
+
+**When in doubt, raise an incident.** We'd much rather have declared an incident which turned out not to be an incident. Many incidents take too long to get called, or are missed completely because someone didn't ring the alarm when they had a suspicion something was wrong.
+
+To declare an incident, type `/incident` in the #incidents channel. This will create a new channel and send updates.
+
+Anyone can declare an incident, including non-engineers. If in doubt, check with your nearest engineer.
+
+Some things that should definitely be an incident
+- `app.posthog.com` being completely unavailable (not just for you)
+- No insights can be created
+- Feature flags are not being returned at all, or `/decide` is down
+- Various alerts defined as critical, such as disk space full, OOM or >5 minute ingestion lag
+
+Things that _shouldn’t_ be an incident
+- Insights returning incorrect data
+- Events being < 5 minutes behind
+- Unable to save insights, create feature flags
```

Comment:
Can you expand why this should not be an incident?

Reply:
Saving insights doesn't stop you from using the insights, and not being able to create feature flags is annoying but doesn't break any existing functionality. Both should be treated as high priority bugs, but aren't incidents